templates/layout/error.php

```diff
@@ -23,7 +23,6 @@
     </title>
     <?= $this->Html->meta('icon') ?>
 
-    <link href="https://fonts.googleapis.com/css?family=Raleway:400,700" rel="stylesheet">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/normalize.css@8.0.1/normalize.css">
```

Comment:
This one is still external right?
So maybe we don't have to remove the railway - if we cant get external resources away anyway.

Reply:
I am not sure why `normalize.css` is used through CDN. It too should be added to webroot/css itself.